lib/jwt/algos/algo_wrapper.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module JWT
+  module Algos
+    class AlgoWrapper
+      attr_reader :alg
+
+      def initialize(alg, cls)
+        @alg = alg
+        @cls = cls
+      end
+
+      def valid_alg?(alg)
+        alg && @alg.casecmp(alg).zero?
```

Comment:
Probably this can be simplified in this way `@alg.casecmp(alg)&.zero?`

Reply:
Good call. I went even one step further and did a `...zero? == true` to ensure the return value always is a boolean